cucumber/protractor-parallel.config.js

```diff
@@ -0,0 +1,42 @@
+"use strict";
+
+const GLOBAL_TIMEOUT = 40e3;
+
+exports.config = {
+  specs: "features/**/*.feature",
+  multiCapabilities: [
+    { browserName: "chrome", shardTestFiles: true, maxInstances: 2 },
+    {
```

Comment:
It is enough to run parallel tests in one browser.

Reply:
fixed